demos/mqtt/mqtt_demo_mutual_auth/mqtt_demo_mutual_auth.c

```diff
@@ -371,11 +371,19 @@ static uint32_t generateRandomNumber();
  * Timeout value will exponentially increase until maximum
  * timeout value is reached or the number of attempts are exhausted.
  *
+ * @param[in,out] pClientSessionPresent Pointer to flag indicating if an
+ * MQTT session is present in the client.
+ * @param[in,out] pBrokerSessionPresent Pointer to flag indicating if a
+ * broker session is present to handle outgoing publishes.
+ * @param[out] pMqttContext The output parameter to return the created mqtt context.
  * @param[out] pNetworkContext The output parameter to return the created network context.
```

Comment:
Parameters should be in order, and some of these descriptions are wrong
```suggestion
 * @param[out] pNetworkContext The output parameter to return the created network context.
 * @param[in] pMqttContext MQTT context pointer.
 * @param[in,out] pClientSessionPresent Pointer to flag indicating if an
 * MQTT session is present in the client.
 * @param[out] pBrokerSessionPresent Session was already present in the broker or not.
 * Session present response is obtained from the CONNACK from broker.
```